compiler/testData/debug/stepping/IfTrueThenFalse.kt

```diff
@@ -0,0 +1,14 @@
+fun cond() = false
+
+fun box() {
+    if (cond())
+        cond()
+    else
+         false
+}
+
+fun main(args: Array<String>) {
+    box()
+}
+
+// 4 1 4 7 8
```

Comment:
Consider also recording the method name and maybe we can do something like:
```
// box(): 4
// cond(): 1
// box(): 4 7 8
```

Reply:
Currently didn't see benefit of adding the function names in test data, @yanex any thoughts?

Reply:
Actually I found it might be helpful, added method information for multi-file use case